### DISCUSSION TITLE
Any suggestions/comments for using Scimap in the analysis of data obtained through Vectra/HALO

### DISCUSSION THREAD

**Discussion by profsprofs1991** [labsyspharm/scimap#29] [Q&A] [answered]
Hi!, 

I am quite new to multiplex imaging analysis overall, and I am trying to use Scimap to perform the anlysis of previous project obtained through mIHC and processed through HALO. I have some questions and would appreciate any commens/suggestions:

1) Cell phenotype was manually assigned in HALO (with output provided as dicotomous (0/1) for each cell). I am currently using this output (0/1 for each channel) to classify each cell with a given phenotype, since  I know `sm.tl.phenotype_cells` uses scaled data and would not be appropiate, but I was wondering if there is a more systematic approach to using a similar function and phenotype strategy. 

2) I see most of the work I have seen in the tutorials have used CyTOF or other "higher" plex methods, is there any limitation of using Scimap for mIHC data (other than the inherit limitations with lower plex capability). 

3) lastly, I would love more about MCMICRO use and Scimap, I was wondering if there is any planned tutorials like the one published that are coming up. 

Thanks a lot in advance, 

Mateo

**Comment by ajitjohnson** [comment, marked as answer]
Hi Mateo,

Happy to help out, glad to see that you would like to try out `scimap`. 

1. If you have already phenotyped your data (each cell has a cell type associated with it), you do not need to use the `sm.tl.phenotype_cells` function. I would just add that as a column in your metadata `.csv` file while importing your data into `scimap`.  I am currently working with some `Vectra` data that has binary calls (similar to yours), so I might add a new function (soon) to take in binary data and perform phenotyping in a similar manner. 

2. I do not anticipate any limitation other than the power to discern finer details of cell states and neighborhoods. 

3. I think @ArtemSokolov can answer that. 

All the best :) And feel free to ask more questions. 

Best,
Ajit.

**Comment by ArtemSokolov** [comment]
3. Actually @hxu-hms is the best person to answer that. ;)

**Comment by hxu-hms** [comment]
Thanks for asking, Mateo! You can find resources for learning MCMICRO on [mcmicro.org](https://mcmicro.org/). If you are new to MCMICRO, we recommend starting with the Tutorial page :) The website also contains information on the modules (including SCIMAP) and provides an example dataset for download to accompany the tutorial. While we do not have plans for a video tutorial now, if it ever becomes available, you will see it on the website!

For supplemental resources, you can find an episode on MCMICRO in our webinar series [here]( https://www.tissue-atlas.org/curriculum).

Feel free to let me know your suggestions on additional materials that will help your learning process!

**Comment by profsprofs1991** [comment]
Thanks a lot everyone for the prompt response, this is very useful!
